_examples/kafka-to-http/router/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bytes"
+	"net/http"
+	"time"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+	watermill_http "github.com/ThreeDotsLabs/watermill/message/infrastructure/http"
+	"github.com/ThreeDotsLabs/watermill/message/infrastructure/kafka"
+	"github.com/ThreeDotsLabs/watermill/message/router/plugin"
+)
+
+var (
+	brokers   = []string{"kafka:9092"}
+	logger    = watermill.NewStdLogger(false, false)
+	topicName = "kafka_to_http_example"
+)
+
+// passMessages passes the message along if its event type is one of acceptedTypes.
+func passMessages(acceptedTypes ...string) message.HandlerFunc {
```

Comment:
maybe `filterMessages` will be better name?

Reply:
ef529080bec252977f397f29a8098a4bbfc6222b